lib/archethic/mining/transaction_context.ex

```diff
@@ -110,9 +100,12 @@ defmodule Archethic.Mining.TransactionContext do
     )
   end
 
-  defp request_utxos(previous_address, nodes) do
+  defp request_utxos(genesis_address, authorized_nodes) do
+    genesis_nodes = Election.chain_storage_nodes(genesis_address, authorized_nodes)
+
     Task.Supervisor.async(TaskSupervisor, fn ->
-      TransactionChain.fetch_unspent_outputs(previous_address, nodes) |> Enum.to_list()
+      TransactionChain.fetch_unspent_outputs(genesis_address, genesis_nodes, true)
```

Comment:
I think we can omit the flag `genesis?`in the  TransactionChain.fetch_unspent_outputs as this function should always load from genesis from now